htdocs/core/lib/process_template.lib.php

```diff
@@ -0,0 +1,58 @@
+<?php
```

Comment:
This is the ajax service i think ?
If i am right, move the file into directory htdocs/core/ajax/emailingemplates.php.